src/saveToCase.js

```diff
@@ -541,7 +532,17 @@ define([
                     });
                 }
                 return $([]);
-            }
+            },
+            getSetValues: function(mug) {
```

Comment:
I think this interface came after save to case was implemented and I missed it in the previous PR. definitely the correct way to do this